src/components/App/routes.tsx

```diff
@@ -15,11 +15,12 @@ import OrderConfirmation from "../../views/OrderConfirmation/View";
 import { ProductPage } from "../../views/Product";
 import SearchPage from "../../views/Search";
 
+const urlRules = ':slug([a-z-0-9]+)/:id([0-9]+)/';
```

Comment:
We could call it `slugUrl` or something like that, so it would make more sense when reading the code.

Reply:
@NyanKiyoshi  renamed to slugUrl